ajax/libs/two.js/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "two.js",
   "filename": "two.min.js",
-  "version": "0.6.0",
+  "version": "0.7.0-beta.1",
```

Comment:
We don't bump non-stable version here.

Reply:
Is it possible to bump non-stable versions anywhere? `v0.6.0` is circa 2016. I'm unable to bump to stable because there is one issue I'm working to resolve with Safari / Webkit team before it's ready.

Reply:
I don't think it really matters, the beta version will still be there, just not the fist appeared one, would that be okay?